qiita_client/plugin.py

```diff
@@ -244,14 +282,33 @@ def __call__(self, server_url, job_id, output_dir):
         with open(self.conf_fp, 'U') as conf_file:
             config.readfp(conf_file)
 
+        # the plugin coupling protocoll can be set in three ways
+        # 1. default is always "filesystem", i.e. first value in
+        #    _ALLOWED_PLUGIN_COUPLINGS. This is to be downward compatible.
+        # 2. the plugin configuration can hold a section 'network' with an
+        #    option 'PLUGINCOUPLING'. For old config files, this might not
+        #    (yet) be the case. Therefore, we are double checking existance
+        #    of this section and parameter here.
+        # 3. you can set the environment variable QIITA_PLUGINCOUPLING
+        # Precedence is 3, 2, 1, i.e. the environment variable overrides the
+        # other two ways.
```

Comment:
Should this be part of the readme in the repo? I think moving these lines there will give them more visibility for future developers; what do you think?